.github/workflows/ci.yml

```
@@ -231,6 +230,20 @@ jobs:
           DB_DATABASE: test
         run: yarn test
 
+      - name: Install playwright
+        working-directory: demo
+        run: yarn playwright install chromium --with-deps --only-shell
+
+      - name: Run external test
+        working-directory: demo
+        env:
+          DB_HOSTNAME: localhost
+          DB_USERNAME: postgres
+          DB_PASSWORD: postgres
+          DB_DATABASE: test
+        run: yarn test:external
+        continue-on-error: true
```

I guess we can let the CI crash on errors if we've fixed all the A11Y issues.